Sources/Apollo/ApolloStore.swift

```diff
@@ -218,6 +213,12 @@ public class ApolloStore {
       accumulator: Accumulator
     ) throws -> Accumulator.FinalResult {
       let object = try loadObject(forKey: key).get()
+      
+      let executor = GraphQLExecutor { object, info in
+        return object[info.cacheKeyForField]
+      } resolveReference: { reference in
+        self.loadObject(forKey: reference.key)
+      }
 
```

Comment:
This is how it was in your main branch. That's where I got the idea to try reverting it. Cleans up the retain cycles 🙌